system/View/Parser.php

```diff
@@ -71,13 +72,8 @@ class Parser extends View
 
     /**
      * Constructor
-     *
-     * @param string          $viewPath
-     * @param mixed           $loader
-     * @param bool            $debug
-     * @param LoggerInterface $logger
      */
-    public function __construct(ViewConfig $config, ?string $viewPath = null, $loader = null, ?bool $debug = null, ?LoggerInterface $logger = null)
+    public function __construct(ViewConfig $config, ?string $viewPath = null, ?FileLocator $loader = null, ?bool $debug = null, ?LoggerInterface $logger = null)
```

Comment:
Changing type declaration is a breaking change. It is not acceptable in the patch release.
Please revert.

Reply:
Done